config/development.json

```diff
@@ -66,6 +66,7 @@
 		"manage/people": true,
 		"manage/people/readers": true,
 		"manage/plans": true,
+		"manage/plan-features": true,
```

Comment:
This is fine for now, but we probably don't want to turn this on for dev until we have a full flow working.